Apps/Src/SendDisplay.c

```diff
@@ -1,7 +1,26 @@
 #include "SendDisplay.h"
-#include "Minions.h"
 #include "MotorController.h"
 #include <math.h>
+#include "Minions.h"
+
+typedef enum {
+     HZD_SW=0, 
+     HEADLIGHT_SW,
+     LEFT_SW,
+     RIGHT_SW,
+} switches_t;
+
+
+typedef enum {  
+    A_CNCTR = 0,
+    M_CNCTR,
+    CTRL_FAULT,
+    LEFT_BLINK,
+    RIGHT_BLINK,
+    Headlight_ON,
+    BrakeLight,
+    RSVD_LED
+} light_t;
```

Comment:
This is for the code to compile. We are not going to rewrite other drivers.

Reply:
If we removed the light and switches enum from the main minions driver, all other references to light_t and switches_t throughout the codebase need to be replaced with MinionPin_t references. Even if the code compiles, it will not function as intended because of the mismatch between enums. Plus, the reason this branch exists is to replace old driver code with new driver calls, which includes using MinionPin_t.

Reply:
Alternatively, just comment out the references to light_t and switches_t enums, as any part of the stack that deals with it is going to get pruned regardless.